laravel/QUICKSTART.md

```diff
@@ -0,0 +1,37 @@
+# Quick Start
+
+## Prerequisite
+
+### Database
+
+> _You are absolutely free to use any UI-based tool, like MySQL Workbench, for creating a new database._
+
+1. `mysql -uroot -p`
+
+   _This command will prompt you to enter the user password. Just hit enter if you've not set a one._
+
+1. `create database awesome_app_db`
+
+## Starting the Laravel app
+
+#### Getting the app up and running
+
+1. `laravel new my-awesome-app`
+1. `cd my-awesome-app`
+1. Open the `.env` file in your favorite code editor and make the following updates:
```

Comment:
I think we will have `.env.example` while creating a new laravel project. We may need to duplicate this one and rename it to `.env`. Please confirm and update if required.

Reply:
I see Laravel added the following scripts, no need to add anything. 
![image](https://user-images.githubusercontent.com/39792899/92310713-9d533980-efce-11ea-89fb-f71bc764a389.png)